src/neomaril_codex/logger_config.py

```diff
@@ -0,0 +1,17 @@
+import sys
+
+from loguru import logger
+
+
+def configure_logger(log_levels=["INFO", "ERROR"]):
+    logger.remove()
+
+    for level in log_levels:
+        logger.add(sys.stdout, level=level, format=f"{{time:MMMM D, YYYY}} | {level}: {{function}} {{message}}")
```

Comment:
Maybe we can have a "preprocess" step that changes the `log_levels` based on an environment variable. This way, it could be useful to still print the debug messages.

We can do it in another PR since this is out of the scope of this current issue.

Reply:
Yeah, I was searching about it, but I couldn't thing on how to implement this. Maybe use a .env variable, but we would need to change the gitignore or try something else (I'm not sure)